src/components/FindProjectTutorial.vue

```diff
@@ -0,0 +1,321 @@
+            console.error('Error fetching tasks for the tutorial', error)
+          },
+          { onlyOnce: true },
+        )
+      }
+    },
+    nextTask() {
+      if (!this.isLastTask) {
+        if (this.isLastTask) {
+          return
+        }
+
+        this.selectedTasks = {}
+        this.currentTaskIndex += 1
+        this.userAttempts = 0
+        this.answersRevealed = false
+      }
+    },
+    updateResult(taskId: string) {
+      if (!this.answersRevealed) {
+        this.userAttempts += 1
+        const currentResult = this.results[taskId]
+
+        if (!isDefined(currentResult)) {
+          this.results[taskId] = this.options[0].value
+          return
+        }
+
+        const newValue = this.optionMap[this.optionMap[currentResult].nextOptionKey].value
+
+        this.results[taskId] = newValue
+      }
+    },
+    handleTileClicked(taskId: string) {
+      this.updateResult(taskId)
+    },
+    handleTileSelected(newValue: boolean, taskId: string) {
+      this.selectedTasks[taskId] = newValue
+    },
+    showAnswer() {
+      this.answersRevealed = true
+      this.taskList.forEach((task) => {
+        this.results[task.taskId] = task.referenceAnswer
+      })
+    },
+  },
+  emits: ['tutorialComplete'],
+  mounted() {
+    goOnline(db)
+    this.fetchTutorialGroups()
+    // this.fetchTutorialProject();
+  },
+})
+</script>
+
+<template>
+  <v-container>
+    <v-row>
+      <v-col>
+        <div>
+          {{ instructionMessage }}
+        </div>
+      </v-col>
+    </v-row>
+    <v-row v-if="alertContent">
+      <v-col>
+        <v-alert
+          density="compact"
+          border="start"
+          variant="tonal"
+          :type="alertContent.type"
+          :title="alertContent.title"
+          :text="alertContent.text"
+          :icon="alertContent.icon"
+        />
+      </v-col>
+    </v-row>
+    <v-row justify="center">
+      <v-col md="6">
+        <div class="image-grid">
+          <find-project-task
+            v-for="task in taskList"
+            :key="task.taskId"
+            :task="task"
+            :optionMap="optionMap"
+            :selected="selectedTasks[task.taskId]"
+            :value="results[task.taskId]"
+            @updateValue="handleTileClicked"
+            @updateSelected="handleTileSelected"
+          />
+        </div>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <task-progress :progress="currentTaskIndex" :total="tutorial?.screens.length ?? 0" />
+      </v-col>
+    </v-row>
+    <v-row justify="center">
+      <v-col cols="auto">
+        <v-btn
+          v-if="userAttempts > 1 && !answeredCorrectly && !answersRevealed"
+          @click="showAnswer"
+        >
+          Show answer
+        </v-btn>
+        <v-btn v-if="!isLastTask && answeredCorrectly" @click="nextTask"> Next task </v-btn>
+        <v-btn v-if="isLastTask && answeredCorrectly" @click="$emit('tutorialComplete')">
+          Start mapping
+        </v-btn>
```

Comment:
Could these buttons be laid over the tile grid, just as in the mobile app, instead of be placed underneath? 

This would reduce vertical space and the need for scrolling while working through the tutorial.

Reply:
@ofr1tz  I've attached these buttons in the alert (as shown in the screenshot below). Let me know if this makes sense!
![2024-10-08-195352](https://github.com/user-attachments/assets/d89db2a7-5f02-4767-9f9c-6fc6bf421689)

Reply:
Makes sense, thanks!